flowmachine/flowmachine/core/query.py

```diff
@@ -537,7 +544,87 @@ def _make_sql(self, name: str, schema: Union[str, None] = None) -> List[str]:
             )
         return queries
 
-    def to_sql(self, name: str, schema: Union[str, None] = None) -> Future:
+    def _unstored_dependencies_graph(self) -> "DiGraph":
```

Comment:
Wondering if these need to live on query, or might be better off in cache.py or their own submodule?

Reply:
Moved into dependency_graph.py, along with all the dependency-graph-related stuff from utils.py